src/org/labkey/test/params/FieldDefinition.java

```diff
@@ -339,7 +341,7 @@ public String getLabel()
 
         public String getRangeURI() { return _rangeURI; }
 
-        protected String getConceptURI()
+        public String getConceptURI()
```

Comment:
`conceptURI` isn't useful outside this class, it just muddies the interface.
```suggestion
        protected String getConceptURI()
```